src/config/config.service.ts

```diff
@@ -82,6 +83,10 @@ export class ConfigService {
     return this.nestConfigService.get<number>('QUEUE_HIGH_WATER')!;
   }
 
+  public getCapacityBatchLimit(): string {
+    return this.nestConfigService.get<string>('CAPACITY_BATCH_LIMIT')!;
+  }
+
```

Comment:
Don't need this injected--can query from the Polkadot-JS API:
`api.consts.frequencyTxPayment.maximumCapacityBatchLength`